### ISSUE TITLE
Packages using `New-Item -Type SymbolicLink` fail

### ISSUE THREAD

**Issue by Schroedingers-Cat** [PietJankbal/Chocolatey-for-wine#4]
Choco packages using the `New-Item -Type SymbolicLink` command fail with `New-Item: No access to target`. Even a simple `New-Item -Type SymbolicLink -Path 'C:\Test' -Value 'C:\Program Files\'` run within `C:\ConEmu\ConEmu64.exe` fails with the same error message.  

Do you have an idea why this isn't working or which software project (wine, conemu, ...) might be causing this?

**Comment by PietJankbal** [comment]
Hi, thanks for reporting this.

I opened a bugreport in wine. It seems a regression in wine-staging. It never worked in wine anyway, but for example the  command you gave works fine in wine-staging-6.14, but fails in current wine-staging-7.9. 

So this has to be (hopefully) fixed in Staging any time;  I`ll report back if so

**Comment by PietJankbal** [comment]
Hi, this is fixed in upcoming Staging-release 7.18 (out in a week or so). Took long time but one of the Staging guys fixed this 2 days ago. So use upcoming Staging and it should be fine.

**Comment by Schroedingers-Cat** [comment]
Finally got around testing this with Wine 8.8. When I open a powershell via `WINEPREFIX=path/to/wineprefix wine powershell.exe` and create a link via `New-Item -Type SymbolicLink -Path 'C:\Test' -Value 'C:\Program Files\'`, the resulting link will be broken. While entering the link works via `cd Test`, you cannot do anything within that directory:
```powershell
PS C:\> dir

    Directory: C:\

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----           10.6.2023    15:59                .REPARSE_POINT
d----           30.5.2023    23:35                ConEmu
d----           10.6.2023    15:10                Program Files
d----           31.5.2023    01:04                Program Files (x86)
d----           31.5.2023    01:09                ProgramData
l----           10.6.2023    15:59                Test -> C:\Program Files\
d----           30.5.2023    23:35                users
d----           30.5.2023    23:58                windows

PS C:\> cd .\Test\
PS C:\Test> dir
Get-ChildItem: Could not find a part of the path 'C:\Test'.
```

Creating a symbolic link from Linux (Terminal, Nautilus, ...) like `ln -s "~/.local/share/WinePrefixes/TestPrefix/drive_c/windows" "~/.local/share/WinePrefixes/TestPrefix/drive_c/working-link"` creates a working link. It is failing to show its target when listed via `dir`, but you can easily cd into the link and operate normally:
```powershell
PS C:\> dir

    Directory: C:\

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----           10.6.2023    15:59                .REPARSE_POINT
d----           30.5.2023    23:35                ConEmu
d----           10.6.2023    15:10                Program Files
d----           31.5.2023    01:04                Program Files (x86)
d----           31.5.2023    01:09                ProgramData
l----           10.6.2023    15:59                Test -> C:\Program Files\
d----           30.5.2023    23:35                users
d----           30.5.2023    23:58                windows
l----           30.5.2023    23:58                working-test ->

PS C:\> cd .\working-link\
PS C:\working-link> dir

    Directory: C:\working-link

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
[...]
-a---           30.5.2023    23:35          63420 twain_32.dll
-a---           30.5.2023    23:35             16 win.ini
-a---           30.5.2023    23:35            257 winhelp.exe
-a---           30.5.2023    23:35         280667 winhlp32.exe
```

Here's the wineprefix root from the native Linux terminal:
```bash
ls -lah
insgesamt 60K
drwxr-xr-x  9 username username 4,0K 10. Jun 16:07  .
drwxr-xr-x  4 username username 4,0K 10. Jun 15:58  ..
drwxr-xr-x  4 username username 4,0K 30. Mai 23:35  ConEmu
drwxr-xr-x 16 username username 4,0K 31. Mai 01:09  ProgramData
drwxr-xr-x 14 username username 4,0K 10. Jun 15:10 'Program Files'
drwxr-xr-x  7 username username 4,0K 31. Mai 01:04 'Program Files (x86)'
drwxr-xr-x  3 username username 4,0K 10. Jun 15:59  .REPARSE_POINT
lrwxrwxrwx  1 username username  158 10. Jun 15:59  Test -> .REPARSE_POINT/Test/./DAAAoFwAAAAAACoALAAiAAAAAABcAD8APwBcAEMAOgBcAFAAcgBvAGcAcgBhAG0AIABGAGkAbABlAHMAXAAAAEMAOgBcAFAAcgBvAGcAcgBhAG0AIABGAGkAbABlAHMAXAAAAA/0
drwxr-xr-x  4 username username 4,0K 30. Mai 23:35  users
drwxr-xr-x 22 username username 4,0K 30. Mai 23:58  windows
lrwxrwxrwx  1 username username   64 10. Jun 16:07  working-link -> /home/username/.local/share/WinePrefixes/TestPrefix/drive_c/windows

```

Maybe it has something to do with the `.REPARSE_POINT` directory, but I'm not sure.

**Comment by PietJankbal** [comment]
This seems to be fixed in wine-staging-9.2. Took a while but it works again afaict.

**Comment by Schroedingers-Cat** [comment]
Thank you so much for reporting this to wine and keeping me posted on this! I can confirm that it's fixed using Wine v9.2. Yay!
